host/sgx/create.c

```diff
@@ -374,6 +374,8 @@ static oe_result_t _add_data_pages(
 }
 
 #if !defined(OEHOSTMR)
+extern const uint32_t supported_cpuid_leaves[OE_CPUID_LEAF_COUNT];
```

Comment:
I'm not sure where this file is getting openenclave/internal/cpuid.h from, but `OE_CPUID_LEAF_COUNT` is defined there and so this file must already have included it for this definition to be valid, as well as the uses of `OE_CPUID_RAX` etc. below.